src/components/AddItems.jsx

```diff
@@ -16,14 +16,33 @@ const radioInputOptions = {
 	notSoon: ['notsoon', nextPurchaseDate.notSoon, 'Not Soon'],
 };
 
-export function AddItems() {
+export function AddItems({ data }) {
 	const [listPath] = useStateWithStorage('tcl-shopping-list-path', null);
+	console.log(data);
```

Comment:
Let's get rid of this console.log before merging.

Reply:
removed